packages/sessions/src/trackers/remote/index.ts

```diff
@@ -56,21 +56,36 @@ export class RemoteConfigTracker implements ConfigTracker, migrator.PresignedMig
     wallet: string
     digest: string
     chainId: ethers.BigNumberish
-    signatures: string[]
+    signatures: string[] | SignerSignature[]
   }): Promise<void> {
     let filteredSignatures = args.signatures
     if (this.onlyRecoverable) {
       filteredSignatures = filteredSignatures.filter(signature => {
-        return commons.signer.canRecover(signature)
-      })
+        if (typeof signature === 'string') {
+          return commons.signer.canRecover(signature)
+        } else {
+          // We "recover" using the included address
```

Comment:
What does this mean?

Reply:
Poorly worded comment. I'll remove it. 
The backend only supports `EIP712` and `ETH_SIGN` types for `SaveSignerSignatures` as it uses recovery, but uses `signature.address` for SaveSignerSignatures2` instead.